docs/api/browser-window.md

```diff
@@ -348,6 +348,9 @@ It creates a new `BrowserWindow` with native properties as set by the `options`.
       You can access this context in the dev tools by selecting the
       'Electron Isolated Context' entry in the combo box at the top of the
       Console tab.
+    * `worldSafeExecuteJavaScript` Boolean (optional) - If true, values returned from `webFrame.executeJavaScript` will be sanitized to ensure JS values
+      can't unsafely cross between worlds when using `contextIsolation`.  The default
+      is `false`. In Electron 12, the default will be changed to `true`. _Deprecated_
```

Comment:
@MarshallOfSound Why is this marked `Deprecated`? I turned this on to prepare for Electron 12, but my code reviewer was confused why I'm using a deprecated option. Presumably `false` is deprecated, but I think we should be explicit about it.

Reply:
@poiru This option is a migration feature, it's kinda weird and you're right we should probably document it better.  The flow here will be

* Add this option
* Change default of this option
* Deprecate this option in code
* Remove this option

We aren't going to leave this flag around forever / for long at all